charts/opensearch-dashboards/README.md

```diff
@@ -74,3 +74,4 @@
 | `tolerations`                      | Configurable [tolerations][]                                                                                                                                                                                                                              | `[]`                                            |
 | `updateStrategy`                   | The [updateStrategy][] for the StatefulSet. By default Kubernetes will wait for the cluster to be green after upgrading each pod. Setting this to `OnDelete` will allow you to manually delete each pod during upgrades                                   | `RollingUpdate`                                 |
 | `extraObjects`                     | Array of extra K8s manifests to deploy                                                                                                                                                                                                                    | list `[]`                                       |
+| `autoscaling`                          | Prerequisite: Install/Configure metrics server. Configurable pod autoscaling stratergy to scale based on `targetCPUUtilizationPercentage`, configure `minReplicas` and `maxReplicas` for desired scaling                                                                                                                                                        | false                             |
```

Comment:
Good to attach the link in the README as well for the installation

Reply:
Fixed in latest commit.